inc/question.class.php

```diff
@@ -261,13 +261,8 @@ private function checkBeforeSave($input) {
                      false,
                      ERROR);
                return array();
-            } else {
-               $input['values'] = addslashes($input['values']);
```

Comment:
duplicates with the method prepareQuestionInputForSave() in the related field types